starter-code/functions-and-arrays.js

```diff
@@ -1,6 +1,10 @@
 // Find the maximum
-function maxOfTwoNumbers (first, second) {
 
+function maxOfTwoNumbers (first, second) {
+  var numList = [first, second];
+  var maxNum = numList.sort().reverse(); //sort the numbers in order and then reverse it
+  maxNum = maxNum[0]; //select the index[0] because it's sorted in descendant order
+  return maxNum;
 }
 
```

Comment:
It seems to complicated. You don't need al those methods. 

function maxOfTwoNumbers (first, second) {
  return first > second ? first : second;
}